src/router.js

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash'
```

Comment:
please import lodash functions like so:
```
import toNumber from 'lodash/toNumber'
import includes from 'lodash/includes'
```

This way only the functionality from lodash is loaded that is actually used